js/bitmax.js

```diff
@@ -135,24 +159,56 @@ module.exports = class bitmax extends Exchange {
         });
     }
 
+    getValidAccounts () {
+        // Bitmax sub-account
+        return ['cash', 'margin', 'futures'];
+    }
+
+    getAccount (params = {}) {
+        // get current or provided bitmax sub-account
+        const account = this.safeValue (params, 'account', this.options['account']);
+        return account.toLowerCase ().capitalize ();
+    }
+
+    setAccount (account) {
+        // set default bitmax sub-account
+        const validAccounts = this.getValidAccounts ();
+        if (account in validAccounts) {
+            this.options['account'] = account;
+        }
+    }
+
+    getFuturesCollateral (params = {}) {
+        // futures collateral
+        const response = this.publicGetFuturesCollateral (params);
```

Comment:
This is missing an `await` and will not work anywhere but Python sync and PHP. It will fail in Python async and JS. I'm going to rewrite this integration completely, since there's too many issues with this file.